src/app/pages/loading-states/loading-states.component.scss

```diff
@@ -0,0 +1,39 @@
+@import '~@pxblue/colors/palette.scss';
+
+mat-list {
+    color: map-get($pxb-gray, 800);
+}
+
+.mat-list-base .mat-list-item {
+    color: inherit;
+    height: 48px;
+    border-top: 1px solid rgba(0, 0, 0, 0.12);
+}
+
+mat-list-item .list-label {
+    margin: 0;
+    margin-left: 15px;
+}
+
+.list-value {
+    flex: 1 1 0px;
+    text-align: right;
+}
+
+.mat-card {
+    width: 100%;
+    overflow: hidden;
+    height: 100%;
+}
+
+.padding0 {
+    padding: 0;
+}
+
+mat-grid-tile {
+    overflow: visible;
+}
+
+::ng-deep .pxb-hero-banner {
```

Comment:
ngdeep plus !important? I don't like seeing these for what's probably a very common tweak that users will want to make on the hero banner.

Reply:
I just replaced the !important with a higher specificity rule.  The ::ng-deep is required because this LoadingState component doesn't use ViewEncapsulation.None.